app/controllers/discussions_controller.rb

```diff
@@ -48,14 +51,25 @@ def update
 
   private
 
+    def check_discussion_is_upcoming
+      if !(@discussion.date.to_s > Date.current.to_s)
```

Comment:
date is a date object. Not a datetime object.

On Fri, Jul 31, 2015 at 4:30 PM, Jaspreet Singh <notifications@github.com>
wrote:

> In app/controllers/discussions_controller.rb
> <https://github.com/SAkshat/iconf/pull/3#discussion_r35964363>:
>
> > @@ -48,14 +51,25 @@ def update
> >
> >    private
> >
> > +    def check_discussion_is_upcoming
> > +      if !(@discussion.date.to_s > Date.current.to_s)
>
> in any case @discussion.date would be a datetime object and so would be
> Date.current. I don't think you need to convert it to string.
> Also, we can govern how what to_s does to date objects. Search it and
> lets discuss.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SAkshat/iconf/pull/3/files#r35964363>.
>



-- 

Akshat Singhal
Vinsol